lib/Sema/SemaDecl.cpp

```diff
@@ -12922,13 +12922,19 @@ void Sema::ActOnBoundsDecl(DeclaratorDecl *D, BoundsAnnotations Annots,
     }
   }
 
+  D->setBoundsExpr(getASTContext(), BoundsExpr);
+  D->setInteropTypeExpr(getASTContext(), IType);
+
   // if this is a member bounds dedclaration, update information mapping
   // members to what bounds declarations depend upon them.
   if (FieldDecl *FD = dyn_cast<FieldDecl>(D))
     TrackMemberBoundsDependences(FD, BoundsExpr);
+  else if (VD && !VD->isLocalVarDeclOrParm()) {
```

Comment:
LLVM code uses "Param" as the abbreviation for parameter. I think we can update "Parm" to "Param". Sounds better.